swag/db.py

```diff
@@ -197,6 +251,63 @@ def creation_date(self, value):
         self.swagdb.creation_date[self.id] = value
 
 
+class Cagnotte:
+    def __init__(self, swagdb, cagnotte_id) -> None:
+        self.swagdb = swagdb
+        self.cagnotte_id = cagnotte_id
+
+    def get_info(self):
+        return CagnotteInfo(self)
+
+    @property
+    def cagnotte_name(self):
+        return self.swagdb.cagnotte_name[self.cagnotte_id]
+
+    @cagnotte_name.setter
+    def cagnotte_name(self, value):
+        self.swagdb.cagnotte_name[self.cagnotte_id] = value
+
+    @property
+    def cagnotte_balance(self):
+        return self.swagdb.cagnotte_balance[self.cagnotte_id]
+
+    @cagnotte_balance.setter
+    def cagnotte_balance(self, value):
+        self.swagdb.cagnotte_balance[self.cagnotte_id] = value
+
+    @property
+    def cagnotte_currency(self):
+        return self.swagdb.cagnotte_currency[self.cagnotte_id]
+
+    @cagnotte_currency.setter
+    def cagnotte_currency(self, value):
+        self.swagdb.cagnotte_currency[self.cagnotte_id] = value
+
+    @property
+    def cagnotte_manager(self):
+        return self.swagdb.cagnotte_manager[self.cagnotte_id]
+
+    @cagnotte_manager.setter
+    def cagnotte_manager(self, value):
+        self.swagdb.cagnotte_manager[self.cagnotte_id] = value
+
+    @property
+    def cagnotte_participant(self):
+        return self.swagdb.cagnotte_participant[self.cagnotte_id]
+
+    @cagnotte_participant.setter
+    def cagnotte_participant(self, value):
+        self.swagdb.cagnotte_participant[self.cagnotte_id] = value
+
+    @property
+    def cagnotte_activation(self):
```

Comment:
`is_active`

Reply:
Well, actually we should not even be able to get a deactivated Cagnotte

Reply:
Resolved in https://github.com/BobbySoftwares/Bybbot/pull/7/commits/5c0172519fd958594b28d5e474628e74e631957e